Dialogflow/src/V2/Gapic/AgentsGapicClient.php

```diff
@@ -702,8 +702,7 @@ public function trainAgent($parent, array $optionalArgs = [])
      *                             Optional.
      *
      *     @type string $agentUri
-     *          Optional. The
-     *          [Google Cloud Storage](https://cloud.google.com/storage/docs/)
+     *          Required. The [Google Cloud Storage](https://cloud.google.com/storage/docs/)
```

Comment:
this flipped from optional to required. the client is pre-GA; could we talk about getting this promoted to the method signature?